app/views/twilio/call.haml

```diff
@@ -1,3 +1,3 @@
 !!! XML
-%Response
-  %Dial= @user.phone
+%response
```

Comment:
I assume haml lint.

But yes, XML is case-sensitive and those tags **must** be capitalized.